index.html

```diff
@@ -12,107 +12,67 @@
-        alt="background icon">
-    </div>
-    <div class="content">
-      <h2>
-        I’m Alexandra.
-        Glad to see you!
-      </h2>
-      <p>
-        I’m a software developer! I can help you build a product,
-        feature or website Look through some of my work and experience!
-        If you like what you see and have a project you need coded,
-        don’t hesitate to contact me.
-      </p>
-      <h4>
-        LET’S CONNECT
-      </h4>
-      <ul>
-        <li>
-          <a href="https://twitter.com/christopheruduekwe">
-            <img src="assets/images/social/bird.png" 
-              alt="angellist icon">
-          </a>
-        </li>
-        <li>
-          <a href="https://www.linkedin.com/in/christopher-uduekwe/">
-            <img src="assets/images/social/Linkedin icon.png" 
-              alt="Linkedin icon">
-          </a>
-        </li>
-        <li>
-          <a href="http://">
-            <img src="assets/images/social/medium.png" 
-              alt="medium icon">
-          </a>
-        </li>
-        <li>
-          <a href="http://github.com/chudisoft">
-            <img src="assets/images/social/github.png" 
-              alt="github icon">
-          </a>
-        </li>
-        <li>
-          <a href="/">
-            <img src="assets/images/social/angellist.png" 
-              alt="angellist icon">
-          </a>
-        </li>
-      </ul>
-    </div>
+    <section>
+      <div class="home-background content">
+        <h1>
+          I’m Alexandra.
+          Glad to see you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded,
+          don’t hesitate to contact me.
+        </p>
+        <h4>
+          LET’S CONNECT
+        </h4>
+        <ul>
+          <li>
+            <a href="https://twitter.com/christopheruduekwe">
+              <img src="assets/images/social/bird.png" 
+                alt="angellist icon">
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/christopher-uduekwe/">
+              <img src="assets/images/social/Linkedin icon.png" 
+                alt="Linkedin icon">
+            </a>
+          </li>
+          <li>
+            <a href="http://">
+              <img src="assets/images/social/medium.png" 
+                alt="medium icon">
+            </a>
+          </li>
+          <li>
+            <a href="http://github.com/chudisoft">
+              <img src="assets/images/social/github.png" 
+                alt="github icon">
+            </a>
+          </li>
+          <li>
+            <a href="/">
+              <img src="assets/images/social/angellist.png" 
+                alt="angellist icon">
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] As per the previous reviewer's suggestion, Kindly apply `flexbox` to align the items inside this section.